Library/Homebrew/dev-cmd/audit.rb

```diff
@@ -1374,6 +1377,15 @@ def audit_urls
       next unless u =~ %r{https?://(?:central|repo\d+)\.maven\.org/maven2/(.+)$}
       problem "#{u} should be `https://search.maven.org/remotecontent?filepath=#{$1}`"
     end
+
+    return unless @online
+    urls.each do |u|
+      begin
+        nostdout { curl "--connect-timeout", "15", "-o", "/dev/null", "-r", "0-0", u }
```

Comment:
Prefer `--output` and `--range` here for readability. I wonder if this is preferable to e.g. a `HEAD` request? Any thoughts?